Resources/Prototypes/_NF/Shipyard/commodore.yml

```diff
@@ -0,0 +1,38 @@
+# Author Info
+# GitHub: crysoskis
+# Discord: crysoskis
+
+# Maintainer Info
+# GitHub: ???
+# Discord: ???
+
+# Shuttle Notes:
+#
+- type: vessel
+  id: Commodore
+  name: NT Commodore
+  description: A small but spacious cargo vessel designed to allow cargo to pass though itself from stern to bow.
+  price: 24500
```

Comment:
So... I have some thoughts about this. The PR description specifically says:

> A beginner-level ship that a first joiner can afford

and

> Intended for pure cargo applications

But a beginner who buys this ship will be left with $500, which isn't enough to haul any kind of cargo anywhere. It's not even enough to buy more plasma! I worry this might end up becoming a noob trap, _especially_ with no bells or whistles at all. How does a first-timer make money off this ship?

Reply:
i only really thought of it as "a fresh player could afford it", in retrospect its not really meant for that anymore

Reply:
you could buy some extra stuff under the original sticker price (not much though), but i had to increase that to fit the 15% minimum requirement